information_retrieval/README.md

```diff
@@ -44,3 +44,19 @@ The included documents are
   used in BM25. BM25 has been shown to be one of the best probabilistic 
   weighting schemes. While the paper was in postscript form, the committer has
   changed the format to pdf as per guidelines of papers we love via ps2pdf.
+
+* [:scroll:](hits.pdf) [Hits Algorithm](https://www.cs.cornell.edu/home/kleinber/auth.pdf) - Jon M. Kleinberg
+
+  This paper introduces the HITS algorithm, a link analysis algorithm that rates 
+  webpages. Unlike the more famous page rank algorithm, the hits algorithm
```

Comment:
Lots of artificial carriage-returns.  I know that we have much of this already, but it would be best to correct the existing while also not add more.  IMO.